packages/vxrn/src/plugins/clientBundleTreeShakePlugin.tsx

```diff
@@ -0,0 +1,99 @@
+import type { Node, Program } from 'estree'
+import { walk } from 'estree-walker'
```

Comment:
we can avoid estree-walker and the complexity, es-module-lexer is way simpler and has less things to handle.